From 6eecc75c169fd9b1a3c26a3c8043f63d4a689d9e Mon Sep 17 00:00:00 2001
Message-Id: <6eecc75c169fd9b1a3c26a3c8043f63d4a689d9e.1561517341.git.aditya@kobol.io>
In-Reply-To: <351f4566aeaff51d999554a2be40ab6a40fb617d.1561517341.git.aditya@kobol.io>
References: <351f4566aeaff51d999554a2be40ab6a40fb617d.1561517341.git.aditya@kobol.io>
From: Aditya Prayoga <aditya@kobol.io>
Date: Wed, 26 Jun 2019 10:48:38 +0800
Subject: [PATCH 2/2] configs: Enable U-Boot RTC support on Helios4

2025-11-23: Tom Urlings: Reapplied build/patch/u-boot/legacy/u-boot-helios4/board_helios4/Add-RTC-to-helios4.patch

---
 configs/helios4_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configs/helios4_defconfig b/configs/helios4_defconfig
index 3fb5f677ce8..f736c790e80 100644
--- a/configs/helios4_defconfig
+++ b/configs/helios4_defconfig
@@ -43,10 +43,11 @@ CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
 CONFIG_CMD_PCI=y
 CONFIG_CMD_SPI=y
 CONFIG_CMD_USB=y
+CONFIG_CMD_DATE=y
 CONFIG_CMD_TFTPPUT=y
 CONFIG_CMD_CACHE=y
 CONFIG_CMD_TIME=y
 CONFIG_ENV_OVERWRITE=y
 CONFIG_ENV_MIN_ENTRIES=128
@@ -70,10 +71,12 @@ CONFIG_PHY_GIGE=y
 CONFIG_MVNETA=y
 CONFIG_MII=y
 CONFIG_MVMDIO=y
 CONFIG_PCI_MVEBU=y
 CONFIG_SPL_DEBUG_UART_BASE=0xd0012000
+CONFIG_DM_RTC=y
+CONFIG_RTC_ARMADA38X=y
 CONFIG_DEBUG_UART_SHIFT=2
 CONFIG_SYS_NS16550=y
 CONFIG_KIRKWOOD_SPI=y
 CONFIG_USB=y
 CONFIG_USB_XHCI_HCD=y
-- 
Created with Armbian build tools https://github.com/armbian/build
